src/modules/storage/BlockStorage.ts

```diff
@@ -81,23 +83,37 @@ export class BlockStorage extends Storages
             return;
         }
 
+        var tx_count: number = 0;
+        var enrollment_count: number = 0;
+
+        if (data.txs)
```

Comment:
Transactions in a block are not optional. So why the `if` check here?

Reply:
That's right. The transactions is required.
Check the error above.

Reply:
> Check the error above.

Will you add the check?

Reply:
Check the block data.txs value. If not, return the error.